docs/protocols.ipynb

```diff
@@ -64,7 +64,7 @@
   },
```

Comment:
<p>nit: "operator" seems like the wrong word.  I would just say "object" since it applies to both operations and gates.  But maybe "operation or gate"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Done.